errors.go

```diff
@@ -243,7 +243,7 @@ const (
 	errTokenInactiveName           = "token_inactive"
 	// errAuthorizaionCodeInactiveName = "authorization_code_inactive"
 	errUnknownErrorName             = "error"
-	errRevokationClientMismatchName = "revokation_client_mismatch"
+	errRevocationClientMismatchName = "revocation_client_mismatch"
```

Comment:
Will this spelling change cause some externally-observable difference?
i.e. does `revocation_client_mismatch` get returned to consumer of this package...

Reply:
Should be fine!